### ISSUE TITLE
Ability to close detail view window

### ISSUE THREAD

**Issue by darbycobb** [fairnessforensics/wiggum#128]
There are some readability issues because you cannot close the detail view after you open it, and it blocks the right side of some of the matrices.
![image](https://user-images.githubusercontent.com/31716469/69159052-f63bfa80-0aac-11ea-9093-6c41dc003542.png)

**Comment by brownsarahm** [comment]
@Shine226 I think it would be nice if the detail view panel could be closed and/or resized (as per #84 ) and the overview matrices could fill the whole space when the detail view is closed. 

At a minimum, the detail view needs to be small enough to not overlap before we can do the userstudy

**Comment by Shine226** [comment]
I made the detail view scalable when the window is resized, it solves the overlap issue.

**Comment by brownsarahm** [comment]
@Shine226 did you commit that directly to master?

**Comment by Shine226** [comment]
I am about making new PR for this if you also think we can close this issue.

**Comment by cegme** [comment]
Looks a lot better, the only strange issue are the appearance (or missing) scroll bars. It is hard to tell whether more scrolling is actually needed. It would also be good to be aligned if possible.
![image](https://user-images.githubusercontent.com/219429/70479328-ba3ff800-1aa2-11ea-9193-a93e4ed442da.png)

**Comment by Shine226** [comment]
The length of the table in the bottom will be changed depending on the datasets. For some datasets, the table may only under the heat map matrix view, but for some datasets, the table will under both matrices view and detail view.

**Comment by Shine226** [comment]
I changed the table size as the same as the matrices view size. 
![Screen Shot 2019-12-09 at 5 13 06 PM](https://user-images.githubusercontent.com/30415315/70481214-86b39c80-1aa7-11ea-8eed-444d0d34f087.png)

**Comment by brownsarahm** [comment]
I think we should avoid horizontal scroll on the table as much as possible. allowing it to be the full width. But, we can make that scroll bar be to the right of the detail plot to make it neater?

Some padding on the detail plot could help with the visual separation, or maybe even lightweight, light colored borders with titles for the sections?

**Comment by Shine226** [comment]
I tried add the borders, but it didn't look great. It is weird that the legend for heatmap is outside of the boarder of the heat map.
![Screen Shot 2019-12-10 at 12 12 59 PM](https://user-images.githubusercontent.com/30415315/70556460-8dddb780-1b46-11ea-9aa9-54fd455dd2fc.png)

**Comment by cegme** [comment]
This looks a lot better to me. You could probably remove the border for the parallel coordinate plot. Maybe add some small internal padding in the main window.

**Comment by Shine226** [comment]
How about this one? Is the internal padding in the matrices view good? The slider for table will go the almost right-most of the window, since the table' width is 78% of the whole window, and margin left for the table is 20%. 
![Screen Shot 2019-12-10 at 1 14 57 PM](https://user-images.githubusercontent.com/30415315/70561026-53c4e380-1b4f-11ea-8cfe-3cf8fcbcf0d2.png)

**Comment by Shine226** [comment]
I adjusted the position for each matrix in the cell. Now it will look like below.
![Screen Shot 2019-12-10 at 1 41 28 PM](https://user-images.githubusercontent.com/30415315/70562920-ee72f180-1b52-11ea-83d0-aaee1590516d.png)
